frankenphp.go

```diff
@@ -698,12 +712,23 @@ func ExecuteScriptCLI(script string, args []string) int {
 	cScript := C.CString(script)
 	defer C.free(unsafe.Pointer(cScript))
 
+	argc, argv := convertArgs(args)
+	defer freeArgs(argv)
+
+	return int(C.frankenphp_execute_script_cli(cScript, argc, (**C.char)(unsafe.Pointer(&argv[0]))))
+}
+
+func convertArgs(args []string) (C.int, []*C.char) {
 	argc := C.int(len(args))
 	argv := make([]*C.char, argc)
 	for i, arg := range args {
 		argv[i] = C.CString(arg)
-		defer C.free(unsafe.Pointer(argv[i]))
```

Comment:
stumbled across this one by accident. If my IDE warning is correct and I understand correctly, `defer` in a for-loop will actually just call `defer C.free(unsafe.Pointer(argv["last-i"]))` i times.

Reply:
😭 good one, the typical for loop bug.